apps/server/src/toasts/toasts.service.ts

```diff
@@ -23,11 +26,21 @@ export class ToastsService {
     });
   }
 
+  getCurrentPeriod(currentDate: Date) {
+    const currentMonth = currentDate.getMonth();
+    const currentYear = currentDate.getFullYear();
+    const startDate =
+      currentMonth >= 0 && currentMonth < 5
+        ? new Date(currentYear, 0, 1)
+        : new Date(currentYear, 6, 1);
```

Comment:
I want you to get rid of these "magic numbers". make a constant that tells the reader what `6` and `0` mean and what the  `1` means